### ISSUE TITLE
PT Lifetime Maximum

### ISSUE THREAD

**Issue by HRAGANBC** [bcgov/SIMS#2694]

**Comment by HRAGANBC** [comment]
@CarlyCotton @Joshua-Lakusta 

I feel we should prioritize this one given potential workload involved. Some questions first:

1. Do we need the last AC with email trigger? Other tickets already capture eCert failures and notifications to student&ministry; my feeling is we could rely on that and don't need seperate trigger type specifically for this type of eCert failure. 
2. Do we want the student's loan balance to be editable by Ministry users through the interface so that when Ministry manually triggers a re-assessment for a student, it can be on against an accurate monthly loan balance instead of whatever was received 29 days ago?
3. Def more detail needed in how the balance gets consumed into the workflows. We landed on simple solution vis-a-vis sequential re-assessments, whereby the monthly loan balance is considered like a 'static' application input rather than a fluid number that alters as a result of sequential re-assessments and new 'room' being created. ...Right?

**Comment by CarlyCotton** [comment]
@HRAGANBC 

1. I don't think we need this specifically for this error type from ecert. We are notifying students and the ministry that there is an error blocking disbursement (though we are less specific with the students and don't include the code like we do for ministry). 
2. Have we checked in with that happens now as a work around on this? I'm not confident on the tolerance of this being accepted/rejected. If we are submitting for a loan <$100 over the LT max, do they just let it through? VS >$500 get hard rejected with no acceptance. I recall there being discussion of tolerance due to the (in)frequent aspect of the report to us.
3. I think that's where we landed, reducing the loan amount based on the total LT loan as of the most recent report. The alternative was doing it at ecert time as it doesn't impact other grant/loan calculations? As far as how this interacts with sequential, we decided that we could not sum up all the loans for the program year to add the monthly figure as we'd be double counting what we disburse

**Comment by HRAGANBC** [comment]
@Joshua-Lakusta Added a mockup, a sample file with dummy SINs, and explicitly listed which fields should be referenced.

**Comment by guru-aot** [comment]
**Questions:** Do we need to delete the file in SFTP after processing?

**Comment by andrewsignori-aot** [comment]
@CarlyCotton is there any appetite to rename the tab name from Overawards to something else since the balances added right now do not seem to fall under the overawrds category?

**Comment by CarlyCotton** [comment]
@andrewsignori-aot Yup, sure is. Though the name we came up with requires us to change another page name. Going to take that away. Thank you.

**Comment by andrewsignori-aot** [comment]
@dheepak-aot can we accommodate the label adjustment for Ministry only in the ticket?
If yes, @CarlyCotton can probably just convert it to a business AC, does it make sense?

**Comment by dheepak-aot** [comment]
@CarlyCotton @andrewsignori-aot I can accommodate that in the ticket.
